autokeras/nn/layers.py

```diff
@@ -300,6 +300,7 @@ def __init__(self,
         super().__init__(input_node, output_node)
         self.kernel_size = kernel_size if kernel_size is not None else Constant.POOLING_KERNEL_SIZE
         self.stride = stride if stride is not None else self.kernel_size
+        # TODO
```

Comment:
What is this to do?

Reply:
for padding, i think. But since we've added the padding, I'll remove it